features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx

```diff
@@ -182,6 +182,7 @@ class Protected extends Component {
             clientId: Utils.getCookieWithoutEnvironment(User.CONST.ADMIN_CLIENT_ID),
             sessionStateCookie: Utils.getCookieWithoutEnvironment(User.CONST.ADMIN_SESSION_STATE),
             mobileOpen: false,
+            isTenant: false,
```

Comment:
Shall we give a proper name to this variable, `isTenant` does not mean anything here right? shouldn't it be `isSuperTenant` ?

Reply:
The API call is to check the tenant information that's why i named that variable isTenant.

Reply:
Fixed with `f01475d`